### ISSUE TITLE
Deck calendars are not synced with CalDav

### ISSUE THREAD

**Issue by Supergamerx3000** [nextcloud/deck#2345]
The automatic deck calendars are not synced to the end devices ios calender app or android calender app.

The calendars are not synchronized with Caldav.

Nextcloud Version 20.0 also tested with nextcloud 19.0.3

Deck APP Version  1.1.0

Calender APP Version 2.1.2

I have tested it under ios with the all other calendars are displayed
also under android
and also in outlook with the caldavsynchronizer no deck calendars are displayed.

Best Regards
Pascal

**Comment by juliushaertl** [comment]
Showing up in calendar depends on if your calendar app supports displaying VTODO entries. I tested this with DAVx5 on android and using the opentasks, though i have no possiblity to test outlook or ios currently. Any help is welcome to figure out why those are not showing up there.

**Comment by Supergamerx3000** [comment]
i sync with opensync on android the opentaskapp show the tasks but when i finish the task and mark the task in the opentask app as done opensync drop a error also davx5.
In addition, I cannot check the task as finished in nextcloud in the online task app

the calender are not show on android with opensync or davx5

on ios i have use the mobileconfig file from nextcloud
and i added the caldav manual in the settings the same result no deck kalender

**Comment by lucker999** [comment]
I can add that tasks show up in Thunderbird, but not much use of it:
- All decks display status as "Need Action" even if the tasks reside in ToDo and Done lists
- No logic to move card to Completed list
- Can't do anything with a task from Thunderbird actually - it doesn't sync back to NC

It would be great to have such features like:
- display only certain lists (and choose which) in a task application (like Thunderbird), for example, only "Doing" list, which make sense, since in backlog could be the whole project, and such a task list would be unmanageable
- sync tags from NC (as Category in Thunderbird for instance) and be able to change them and sync changes back
- marking a task completed moves it to "Done" list.

This is a bare minimum to make Dack app usable with another app, on a mobile todo list, for example. Alas, seems like it wouldn't be done.

**Comment by hansimausisteffiwurstbrot** [comment]
Hey, as this is a very basic feature:
Is anyone willing to solve this issue? We'd be glad to support it with 200 €.

**Comment by Noizecube** [comment]
I am also interested in Solving this issue!

**Comment by stefan-niedermann** [comment]
@Noizecube Please use **[GitHub reactions](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/)** 👍 to show that you are affected by the same issue. Commenting without adding new information will spam everyone who subscribed and makes it hard to follow the history of an issue.

**Comment by janvlug** [comment]
I am using Evolution on Fedora 35. The Deck task list do show up in Evolution, but an error message notification pops up when Evolution tries to synchronize them. See also: https://github.com/nextcloud/deck/issues/2345.

**Comment by fightingminers** [comment]
Please! I really love the simplicity of this app! Can you please just make the syncing to outlook/ ios work? I have some users on my job who like the online tool in nextcloud but also need the sync to the calendar in both directions. If you make it possible i will donate 500€ to you. Maybe rethink the way you transfer the data of the canban system into the calendar - maybe a helper syncing internally to a simple calendar task would fix it? Thanks a lot!

**Comment by wunschadresse** [comment]
Wow, there is quite a bounty on this topic. Hope it will be implemented soon.
What i have found is that davx5 does sync the decks with android but rather rather as tasks, not as calendars. Should be an easy fixxer upper, no?

**Comment by rightsaidfred99** [comment]
Can confirm the private link address doesn't work with macos' calendar either. It's just blank

**Comment by 31337-4554551n** [comment]
Just showing support and interest for this.
I would love to see my cards from deck with due dates in etar on android.
Currently they can only be shown (via davx5) in tasks apps. But it's so much better in one pane of glass, how the nextcloud webui has it.

Hoping someone can take a look, showing my support that interest in this still exists.

**Comment by hansimausisteffiwurstbrot** [comment]
There ist a workaround by using Tasks.org and syncing the Deck via this app...

https://f-droid.org/en/packages/org.tasks/

But imho it doesn't solve this issue.

**Comment by trougnouf** [comment]
> There ist a workaround by using Tasks.org and syncing the Deck via this app...
> 
> https://f-droid.org/en/packages/org.tasks/
> 
> But imho it doesn't solve this issue.

(How) Does that solve 2-way synchronization?
